src/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+  <title>Wbpack Exercise</title>
+</head>
+<body>
+  <div class="container">
```

Comment:
**Changes Required**
- [ ] The `index.html` should only contain the root element, i.e `.container`. The rest of the html should be generated dynamically by manipulating the `DOM` using JavaScript. You could checkout [this freecodecamp](https://www.freecodecamp.org/news/how-to-manipulate-the-dom-beginners-guide/) article plus any other web resources in line with this subject. Please let us refactor this accordingly.